kalliope/core/OrderAnalyser.py

```diff
@@ -50,15 +53,36 @@ def get_matching_synapse(cls, order, brain=None):
 
         # test each synapse from the brain
         for synapse in cls.brain.synapses:
-            # we are only concerned by synapse with a order type of signal
             for signal in synapse.signals:
-
+                # we are only concerned by synapse with a order type of signal
                 if signal.name == "order":
-                    if cls.spelt_order_match_brain_order_via_table(signal.parameters, order):
+                    # get the type of matching expected, by default "normal"
+                    expected_matching_type = "normal"
+                    signal_order = None
+
+                    if isinstance(signal.parameters, str):
+                        signal_order = signal.parameters
+                    if isinstance(signal.parameters, dict):
+                        try:
+                            signal_order = signal.parameters["text"]
+                        except KeyError:
+                            logger.debug("[OrderAnalyser] Warning, missing parameter 'text' in order. "
+                                         "Order will be skipped")
+                            continue
+                        try:
+                            expected_matching_type = signal.parameters["matching-type"]
+                        except KeyError:
+                            logger.debug("[OrderAnalyser] Warning, missing parameter 'matching-type' in order. "
+                                         "Fallback to 'normal'")
+                            expected_matching_type = "normal"
```

Comment:
expected_matching_type is already set up to normal : https://github.com/kalliope-project/kalliope/pull/398/files#diff-cf477bc667a930aea9ea3b8f49b702c2R60